ansible/server-state-playbooks/ome-dundeeomero/playbook.yml

```diff
@@ -135,7 +135,8 @@
     # Note - had to have these set to `install-mock` to progress role
     # installation before changing config to restored DB from other system.
     - role: openmicroscopy.omero-server
-      omero_server_release: 5.3.1
+      omero_server_release: 5.3.2
+      omero_server_upgrade: True
```

Comment:
Ah, that'd be nice if it quashed all the `restart systemd` calls. Need to get to know the playbook more. That was one of my lingering questions about re-running - i.e. I'm not sure I'd be happy rerunning this outside downtime, and therefore wouldn't straight off want it running on a daily cron.